egs/mucs21_subtask2/asr1/local/gen_wavscp.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Could you briefly explain what this script does?

Reply:
> Cool.
> I think we can merge this PR and we can have a follow-up PR to include RESULTS.ms and model link.
> If you fix the CI error, I can merge this PR.

Okay Sure. Will do it in the morning.